readme.md

```diff
@@ -1,3 +1,5 @@
+![auto-annotate-logo](https://raw.githubusercontent.com/Lucs1590/auto_annotate/master/images/logo.png)
```

Comment:
@AlvaroCavalcante  you may need to change the link, but accept the pull request first.

Reply:
@Lucs1590 Very nice logo, thank you, my friend! :)